Assets/Events/Flavour.json

```diff
@@ -16,7 +16,7 @@
   {
     "name": "esquilax",
     "headline": "Fabled Beast Spotted Nearby",
-    "article": "Witnesses claim to have seen the mythical Esquilax!. Based on our readings of <i>\"The Big Book of Beasts\"</i>, it has been described as <i>\"A horse, with the head of a rabbit, and the body, of a rabbit.\"</i>",
+    "article": "Witnesses claim to have seen the mythical Esquilax!. Based on our readings of <i>\"The Big Book of Beasts\"</i>, it has been described as <i>\"A horse, with the head of a rabbit and the body of a rabbit.\"</i>",
```

Comment:
I think this joke needs some pacing to be funny, a way to break up the head and body part. Anything else we can use instead of the commas, if we don't like that grammer?

Reply:
It works better when it's ready quickly anyway. Otherwise it errs on the side of campy.